src/algebra/ordered_ring.lean

```diff
@@ -138,9 +138,11 @@ mul_le_mul_of_nonneg_right h hb
 
 lemma add_le_mul_two_add [nontrivial α] {a b : α}
   (a2 : 2 ≤ a) (b0 : 0 ≤ b) : a + (2 + b) ≤ a * (2 + b) :=
-(((add_le_add_left (add_le_add a2 (le_mul_of_one_le_left b0 (one_le_two.trans a2))) a).trans
-  (le_of_eq (add_assoc a a _).symm)).trans (add_le_add_right (le_of_eq (mul_two a).symm) _)).trans
-  (le_of_eq (mul_add a 2 b).symm)
+calc a + (2 + b) ≤ a + (a + a * b) :
+      add_le_add_left (add_le_add a2 (le_mul_of_one_le_left b0 (one_le_two.trans a2))) a
+             ... ≤ a * (2 + b) :
+      le_trans (le_trans (le_of_eq (add_assoc a a _).symm) (le_trans rfl.ge (add_le_add_right
```

Comment:
I'm not sure this term-mode proof is worth it over the triplet of rewrites I suggested before

Reply:
Ok, I will push your version with rewrites!